hack/addons/terraform/terraform-provider-scaffold/metadata.yaml

```diff
@@ -17,22 +17,15 @@ dependencies:
 
 # -------------------------------------Configuration Metadata for a Terraform Addon-------------------------------------
 # provider short name
-shortCloudName: tencent
+shortCloudName: elastic
 
 # The Cloud name of the provider
-completeCloudName: Tencent Cloud
+completeCloudName: Elastic Cloud
 
 # When enabling a Terraform provider, these properties need to set for authentication. For Tencent Cloud,
 # name: Environment variable name when authenticating Terraform, like https://github.com/oam-dev/terraform-controller/blob/master/controllers/provider/credentials.go#L59
 # secretKey: Secret key when storing authentication information in a Kubernetes, like https://github.com/oam-dev/terraform-controller/blob/master/controllers/provider/credentials.go#L109.
 cloudProperties:
-  - name: TENCENTCLOUD_SECRET_ID
```

Comment:
This file should not be touched as you have used hack/addons/terraform/terraform-provider-scaffold/metadata.yaml as the meta file.

Reply:
You mean set the metadata.yaml back to the previous state after I use it to generate elastic cloud provider?

Reply:
Do you need to modify this doc by We have a new way to generate a Terraform addon, please refer to https://kubevela.net/docs/platform-engineers/addon/terraform?